task/host-sp-comms/src/main.rs

```diff
@@ -1025,6 +1063,16 @@ impl idl::InOrderHostSpCommsImpl for ServerImpl {
     ) -> Result<Status, RequestError<HostSpCommsError>> {
         Ok(self.status)
     }
+
+    fn get_inventory(
```

Comment:
I'm not sure about this idol op - was this a debugging/development aid? If some other task (or even humility) calls this while we're servicing a request from the host, things will get pretty confused, I think.

Reply:
This was used for debugging (to check stack sizes); I thought that leaving it was innocuous, but you make a good point!  Removed in e83fa92dd0b8b4986f2ec94f50d2e4c38a4e8a36